packages/neo-one-client-common/src/crypto.ts

```diff
@@ -575,6 +586,192 @@ const isSignatureContract = (script: Buffer) => script.length === 35 && script[0
 
 const isStandardContract = (script: Buffer) => isSignatureContract(script) || isMultiSigContract(script);
 
+const HARDENED_KEY_OFFSET = 0x80000000;
+const EXTENDED_KEY_BYTES = 78;
+
+// Need eyes, according to https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki#node-serialization this isn't special.
+const BIP32_VERSION = {
+  public: 0x0488b21e,
+  private: 0x0488ade4,
+};
+
+export interface HDNode {
+  readonly privateKey?: PrivateKey;
+  readonly publicKey: Buffer;
+  readonly version: number;
+  readonly depth: number;
+  readonly parentFingerprint?: Buffer;
+  readonly index: number;
+  readonly chainCode: Buffer;
+}
+
+const getFingerprint = (value: Buffer) => hash160(value).slice(0, 4);
+
+const parseMasterSeed = (seedIn: Buffer | string): HDNode => {
+  const seed = seedIn instanceof Buffer ? seedIn : Buffer.from(seedIn, 'hex');
+  const hmac = hmacSha512('NEO Seed', seed);
```

Comment:
Why `'NEO Seed'`?

Reply:
The seed name doesn't matter. I only copied this from 'BTC Seed' and changed it to `NEO`.